whole-tale/terraform_deployment
# JS2 deployment fixes

This PR: 
* Adds NFS support for use with https://github.com/whole-tale/gwvolman/pull/172
* Updates variables.tf defaults to JS2

To test:
* Provision new cluster or use current js2.wholetale.org
* Confirm mounts are working as expected
